corehq/apps/app_manager/suite_xml.py

```diff
@@ -920,8 +926,63 @@ def details(self):
 
                         if detail_column_infos:
                             if detail.custom_xml:
-                                d = load_xmlobject_from_string(detail.custom_xml, xmlclass=Detail)
+                                d = load_xmlobject_from_string(
+                                    detail.custom_xml,
+                                    xmlclass=Detail
+                                )
                                 r.append(d)
+
+                            elif detail.use_case_tiles:
+
+                                template_args = {
+                                    "detail_id": self.id_strings.detail(module, detail_type),
+                                    "title_text_id": self.id_strings.detail_title_locale(
+                                        module, detail_type
+                                    )
+                                }
+
+                                # Get field/case property mappings
+                                for template_field in ["header", "top_left", "sex", "bottom_left", "date"]:
+                                    column = None
+                                    for col in detail.columns:
+                                        if col.case_tile_field == template_field:
+                                            column = col
+                                            break
```

Comment:
It would be more efficient these detail columns once making a dict, and then use that to look up each template field:
```python
cols_by_tile = {col.case_tile_field: col for col in detail.columns}
for template_field in ...
    column = cols_by_tyle.get(template_field)
```